src/main/java/com/task/api/services/OrderService.java

```diff
@@ -0,0 +1,18 @@
+package com.task.api.services;
+
+import com.task.api.dto.BaseDTO;
+import io.restassured.http.Header;
+import io.restassured.response.Response;
+
+public class OrderService extends BaseService {
+    private static final String CHECKOUT_PATH = "/api/CheckOut/{userId}";
+    private static final String GET_ALL_ORDERS_PATH = "/api/Order/{userId}";
+
+    public Response checkout(Header header, BaseDTO body, Object... vars) {

```

Comment:
passing Object... vars is too vague and can be a reason of mistakes, should be changed to some king of request body
```suggestion
    public Response checkout(Header header, BaseDTO body, CheckoutRequestDto requestBody) {

```